src/utils/changes.ts

```diff
@@ -175,3 +181,152 @@ export function prependChangeset(
 
   return markdown.slice(0, startIdx) + newSection + markdown.slice(startIdx);
 }
+
+interface Commit {
+  hash: string;
+  title: string;
+  hasPRinTitle: boolean;
+  pr: string | null;
```

Comment:
If there's a `pr` field, why do we have another field saying whether there's a PR? If there isn't, `pr === null`; if there is, `pr` will be a string, right?

Reply:
It says hasPRin**Title** not just "hasPR" 😁 Maybe I can add a comment here:

Typically, GitHub automatically adds the PR number at the end of commit title in the format ` (#<PR_NO>)`. This allows us to infer the PR number from the commit log only, staying offline. However, it is possible to remove this line when merging PRs. Even more, it is very easy to just use the commit SHA to get from a commit to a pr to a milestone using GraphQL. This only matters when formatting the commit (or potentially if a commit is associated with multiple PRs but that's irrelevant for us for now) as we want to know whether we should append the PR number provided by GitHub at the end of the commit title or not.

Makes sense?

Reply:
Ooops 😅